WalletWasabi/WabiSabi/Backend/Rounds/Round.cs

```diff
@@ -140,14 +140,22 @@ public SigningState AddWitness(int index, WitScript witness)
 			=> Assert<SigningState>().AddWitness(index, witness);
 
 		private uint256 CalculateHash()
-			=> StrobeHasher.Create(ProtocolConstants.RoundStrobeDomain)
-				.Append(ProtocolConstants.RoundMinRegistrableAmountStrobeLabel, MinRegistrableAmount)
-				.Append(ProtocolConstants.RoundMaxRegistrableAmountStrobeLabel, MaxRegistrableAmount)
-				.Append(ProtocolConstants.RoundMaxRegistrableVsizeStrobeLabel, MaxRegistrableVsize)
-				.Append(ProtocolConstants.RoundMaxVsizePerAliceStrobeLabel, MaxVsizeAllocationPerAlice)
-				.Append(ProtocolConstants.RoundAmountCredentialIssuerParametersStrobeLabel, AmountCredentialIssuerParameters)
-				.Append(ProtocolConstants.RoundVsizeCredentialIssuerParametersStrobeLabel, VsizeCredentialIssuerParameters)
-				.Append(ProtocolConstants.RoundFeeRateStrobeLabel, FeeRate.FeePerK)
-				.GetHash();
+			=> RoundHasher.CalculateHash(
+					InputRegistrationTimeout,
+					ConnectionConfirmationTimeout,
+					OutputRegistrationTimeout,
+					TransactionSigningTimeout,
+					CoinjoinState.Parameters.AllowedInputAmounts,
+					CoinjoinState.Parameters.AllowedInputTypes,
+					CoinjoinState.Parameters.AllowedOutputAmounts,
+					CoinjoinState.Parameters.AllowedOutputTypes,
+					CoinjoinState.Parameters.Network,
+					CoinjoinState.Parameters.FeeRate.FeePerK,
+					CoinjoinState.Parameters.MaxTransactionSize,
+					CoinjoinState.Parameters.MinRelayTxFee.FeePerK,
+					MaxRegistrableVsize,
```

Comment:
Sorry, I misled you, although `MinRegistrableAmount` is redundant with `Allowed{In,Out}putAmounts.Min`, `MaxRegistrableAmount` still needs to be committed, because the server still serializes it and the client uses it to compute the range proof width independently of `Allowed{In,Out}put.Max`, which only constrain the blockchain level values. Without this clients may be tricked into submitting range proofs of different widths, allowing them to be differentiated by a malicious coordinator.

```suggestion
					MaxRegistrableAmount,
					MaxRegistrableVsize,
```

Reply:
The server already commits to this and the client computes the round hash independently so, there is not way the coordinator can trick the client, otherwise the hash would not match.